Insterm/SSHTerminalView.swift

```diff
@@ -38,6 +39,28 @@ class SSHTerminalView: TerminalView, TerminalViewDelegate, NMSSHChannelDelegate
         print("didReadError: \(error)")
     }
     
+    func isConnected() -> Bool {
+        if (connection != nil) {
```

Comment:
optinal のforce unrwapはクラッシュの原因になり、コードの品質を落とすので、基本的には使わないようにします。
今回であればnil coalescingを使って以下のように書けるかと思います。

```suggestion
        return connection?.isConnected() ?? false
```